quickwit/quickwit-indexing/src/actors/packager.rs

```diff
@@ -134,14 +134,6 @@ impl Handler<IndexedSplitBatch> for Packager {
             split_ids=?split_ids,
             "start-packaging-splits"
         );
-        for split in &batch.splits {
-            if let Some(controlled_directory) = &split.controlled_directory_opt {
```

Comment:
index serialization used to happen here, which is why we took over the controlled directory in the packager.

Today the serialization logic was moved int he IndexSerializer so I moved these lines to the IndexSerializer.